mobile-app/app/screens/AppNavigator/screens/Auctions/components/EmptyAuction.tsx

```diff
@@ -1,56 +1,46 @@
 import { tailwind } from "@tailwind";
-import { ThemedIcon, ThemedText, ThemedView } from "@components/themed";
-import { translate } from "@translations";
-import { InfoTextLink } from "@components/InfoTextLink";
-import { View } from "react-native";
+import { InfoTextLinkV2 } from "@components/InfoTextLink";
 import { useNavigation, NavigationProp } from "@react-navigation/native";
+import { translate } from "@translations";
+import { useThemeContext } from "@shared-contexts/ThemeProvider";
 import { AuctionsParamList } from "../AuctionNavigator";
+import { EmptyTokensScreen } from "../../Portfolio/components/EmptyTokensScreen";
+import { EmptyAuctionsDark } from "../../Portfolio/assets/EmptyAuctionsDark";
+import { EmptyAuctionsLight } from "../../Portfolio/assets/EmptyAuctionsLight";
 
-export function EmptyAuction(): JSX.Element {
+export function EmptyAuction({
+  title,
+  subTitle,
```

Comment:
subtitle is actually one word
```suggestion
  subtitle,
```